src/main/webapp/WEB-INF/tags/navBarBrand.tag

```diff
@@ -0,0 +1,8 @@
+<%@ tag description="TEAMMATES logo brand"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
+<%@ tag import="teammates.common.util.Const" %>
```

Comment:
Just noticed: this may not be necessary (does it work without)? The previous two lines it above too.